src/main/java/mods/eln/item/electricalitem/ElectricalLampItem.kt

```diff
@@ -73,7 +73,7 @@ class ElectricalLampItem(name: String, var lightMin: Int, var rangeMin: Int, dis
     }
 
     override fun onItemRightClick(s: ItemStack, w: World, p: EntityPlayer): ItemStack {
-        if (!w.isRemote) {
```

Comment:
I'm not sure this is right. It prevents setLightState from being called for the correct state (off) when off. It might be handled elsewhere as well but should we make sure it's turned off (and tell them it's off) when they click?